app/controllers/pet_applications_controller.rb

```diff
@@ -20,6 +20,10 @@ def update
       application.update!({
         status: "Approved"
     })
+    else application.all_pet_statuses.any?("Rejected")
```

Comment:
Now that this method is in the pet_applications_controller vs. applications_controller it makes sense why it would need to be there. The joins table should be routing back to the applications because applications can't see a pet's status without going through pet_applications